internal/server/storage/memstorage.go

```diff
@@ -0,0 +1,83 @@
+package storage
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+)
+
+type MemStorage struct {
+	data map[string]string
```

Comment:
It would be better if it's `interface{}` or `any`. But it's not an efficient solution